sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/src/Helpers.cs

```diff
@@ -47,9 +43,9 @@ internal static EventDefinition GetEventDefintionFromPayload(string payload)
 
                 throw new Exception(string.Format(CultureInfo.CurrentCulture, AuthenticationEventResource.Ex_Comparable_Not_Found, comparable));
             }
-            catch (Exception ex)
+            catch (Exception)
```

Comment:
```suggestion
            catch
```

Is there really no way to tune this to a more specific error?   My concern is that any unexpected error gets wrapped up and thrown away.   Context for things that you don't expect is lost.

Reply:
Good point. @ShaneMicro, I am thinking of putting the the exception at minimum as a inner exception.